src/metal/rt64_metal.h

```diff
@@ -3,6 +3,7 @@
 #include <set>
 
 #include "rhi/rt64_render_interface.h"
+#include "common/rt64_apple.h"
```

Comment:
Plume won't have access to this, so we should integrate it as part of `rt64_metal.h`.

Reply:
Which means `rt64_apple.mm` will probably need to be migrated to `rt64_metal.mm`.